### ISSUE TITLE
metallb ImageInspectError

### ISSUE THREAD

**Issue by Aoana** [Nordix/xcluster#2]
When installing metallb (not using Helm) I get an ImageInspectError.
The installation of metallb are from their page: [metallb installation](https://metallb.universe.tf/installation/)

### Steps to reproduce

```
[xcluster] $ source Envsettings
"diskim" is missing. Install with;

wget -O - -q \
 https://github.com/lgekman/diskim/releases/download/v0.4.0/diskim-v0.4.0.tar.xz \
 | tar -I pxz -C /home/eansmal/xcluster/workspace -xf -

[xcluster] $ source Envsettings.k8s 
[xcluster] $ xc mkcdrom externalip
00externalip.tar
[xcluster] $ xc start
[xcluster] $ vm 1

vm-001 ~ # kubectl apply -f https://raw.githubusercontent.com/google/metallb/v0.
7.3/manifests/metallb.yaml
namespace/metallb-system created
serviceaccount/controller created
serviceaccount/speaker created
clusterrole.rbac.authorization.k8s.io/metallb-system:controller created
clusterrole.rbac.authorization.k8s.io/metallb-system:speaker created
role.rbac.authorization.k8s.io/config-watcher created
clusterrolebinding.rbac.authorization.k8s.io/metallb-system:controller created
clusterrolebinding.rbac.authorization.k8s.io/metallb-system:speaker created
rolebinding.rbac.authorization.k8s.io/config-watcher created
daemonset.apps/speaker created
deployment.apps/controller created
vm-001 ~ # kubectl get pods -n metallb-system
NAME                         READY   STATUS              RESTARTS   AGE
controller-765899887-7kd84   0/1     ImageInspectError   0          12s
speaker-6s7pt                0/1     ImageInspectError   0          12s
speaker-6tddc                0/1     ImageInspectError   0          12s
speaker-dn2kd                0/1     ImageInspectError   0          12s
speaker-nn8z9                0/1     ImageInspectError   0          12s
```
### Additional Information

```
vm-001 ~ # kubectl describe pod controller-765899887-7kd84 -n metallb-system
Name:               controller-765899887-7kd84
Namespace:          metallb-system
Priority:           0
PriorityClassName:  <none>
Node:               vm-001/192.168.0.1
Start Time:         Fri, 05 Oct 2018 07:32:03 +0000
Labels:             app=metallb
                    component=controller
                    pod-template-hash=765899887
Annotations:        prometheus.io/port: 7472
                    prometheus.io/scrape: true
Status:             Pending
IP:                 11.0.1.2
Controlled By:      ReplicaSet/controller-765899887
Containers:
  controller:
    Container ID:  
    Image:         metallb/controller:v0.7.3
    Image ID:      
    Port:          7472/TCP
    Host Port:     0/TCP
    Args:
      --port=7472
      --config=config
    State:          Waiting
      Reason:       ImageInspectError
    Ready:          False
    Restart Count:  0
    Limits:
      cpu:     100m
      memory:  100Mi
    Requests:
      cpu:        100m
      memory:     100Mi
    Environment:  <none>
    Mounts:
      /var/run/secrets/kubernetes.io/serviceaccount from controller-token-69vjc 
Conditions:
  Type              Status
  Initialized       True 
  Ready             False 
  ContainersReady   False 
  PodScheduled      True 
Volumes:
  controller-token-69vjc:
    Type:        Secret (a volume populated by a Secret)
    SecretName:  controller-token-69vjc
    Optional:    false
QoS Class:       Guaranteed
Node-Selectors:  <none>
Tolerations:     node.kubernetes.io/not-ready:NoExecute for 300s
                 node.kubernetes.io/unreachable:NoExecute for 300s
Events:
  Type     Reason             Age                   From               Message
  ----     ------             ----                  ----               -------
  Normal   Scheduled          14m                   default-scheduler  Successfully assigned metallb-system/controller-765899887-7kd84 to vm-001
  Warning  InspectFailed      13m (x8 over 14m)     kubelet, vm-001    Failed to inspect image "metallb/controller:v0.7.3": rpc error: code = Unknown desc = no registries configured while trying to pull an unqualified image, add at least one in /etc/crio/crio.conf under the "registries" key
  Warning  Failed             13m (x8 over 14m)     kubelet, vm-001    Error: ImageInspectError
  Warning  MissingClusterDNS  4m46s (x49 over 14m)  kubelet, vm-001    pod: "controller-765899887-7kd84_metallb-system(c094b7a7-c870-11e8-a964-000000010001)". kubelet does not have ClusterDNS IP configured and cannot create Pod using "ClusterFirst" policy. Falling back to "Default" policy.
```
Is this issue possibly related? [clearlinux distribution issue](https://github.com/clearlinux/distribution/issues/160)

**Comment by uablrek** [comment]
All explained here; https://medium.com/cri-o/default-container-registry-revisited-50b32dd808fe

**Comment by uablrek** [comment]
A fix is committed and will be in the next binary release. As a work-around download and include the attached ovl. Unpack somewhere in your `$XCLUSTER_OVLPATH` and add `issue-2` when you do `xc mkcdrom`.

[ovl-issue-2.tar.gz](https://github.com/Nordix/xcluster/files/2449635/ovl-issue-2.tar.gz)

**Comment by uablrek** [comment]
Released in v0.2
